src/NServiceBus.RavenDB.Tests/SubscriptionStorage/When_receiving_duplicate_subscription_messages.cs

```diff
@@ -32,7 +32,27 @@ public class When_receiving_duplicate_subscription_messages : RavenDBPersistence
         }
     }
 
- 
+    [Test]
+    public async Task should_not_create_additional_db_rows_even_if_event_version_changed()
```

Comment:
that opens an interesting question: is the `Endpoint` a key identifier or not? Or is the `address` that identifies the subscriber? Or both in combination?